src/modules/person/api/person.controller.ts

```diff
@@ -441,6 +444,30 @@ export class PersonController {
         return new PersonLockResponse(`User has been successfully ${lockUserBodyParams.lock ? '' : 'un'}locked.`);
     }
 
+    @Post(':personId/sync')
+    @HttpCode(HttpStatus.OK)
+    @ApiOkResponse({ description: 'User will be synced.' })
+    @ApiNotFoundResponse({ description: 'The person was not found.' })
+    @ApiForbiddenResponse({ description: 'Insufficient permissions to perform operation.' })
+    @ApiInternalServerErrorResponse({ description: 'An internal server error occurred.' })
+    @ApiBadGatewayResponse({ description: 'A downstream server returned an error.' })
+    public async syncPerson(
+        @Param('personId') personId: string,
+        @Permissions() permissions: PersonPermissions,
+    ): Promise<void> {
+        const personResult: Result<Person<true>> = await this.personRepository.getPersonIfAllowed(
+            personId,
+            permissions,
+            // TODO SPSH-1136 Update the RollenSystemRecht
+            [RollenSystemRecht.PERSONEN_VERWALTEN],
+        );
+        if (!personResult.ok) {
+            throw new NotFoundOrNoPermissionError(personId);
```

Comment:
Is it not better to distinguish errors here? Maybe return a not found if findById for Person doesn't find anything and NoPermission if the admin has no permission to find the user?

Reply:
If we differentiate, a user could figure out if a person with a specific ID exists. Of course just knowing that an ID exists is not really "sensitive" information, but we do this in multiple places to not leak the information